### ISSUE TITLE
RunPod 3.2.0: webui.sh: line 255: 146 Illegal instruction

### ISSUE THREAD

**Issue by kozhemyak** [ashleykleynhans/stable-diffusion-docker#23]
Hi!

First of all, thank you for this amazing docker image. I use it very often within the RunPod instance.

I encountered an issue deploying a fresh instance. I am unsure if it is related to the image, but I rely on your help to solve it.
Unfortunately, my attempts were unsuccessful.

When I deploy a new instance on RunPod, after starting Automatic1111 it doesn't start with the following error:
```
...
################################################################
Launching launch.py...
################################################################
Using TCMalloc: libtcmalloc_minimal.so.4
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
Version: v1.6.0
Commit hash: 5ef669de080814067961f28357256e8fe27544f4
Launching Web UI with arguments: -f --port 3001 --skip-install --listen --api --xformers --enable-insecure-extension-access --no-half-vae
./webui.sh: line 255:   146 Illegal instruction     (core dumped) "${python_cmd}" -u "${LAUNCH_SCRIPT}" "$@"
```

Thank you for any help in advance.

**Comment by ashleykleynhans** [comment]
There is an issue with the pod, create a new one.

**Comment by kozhemyak** [comment]
Yes, i did it multiple times. All the time, I have the same issue. I will try to use another region.

**Comment by ashleykleynhans** [comment]
Did you have any luck with a different region?

**Comment by ashleykleynhans** [comment]
Seems the hosts in the FR region in Secure Cloud are broken, I asked RunPod to either remove or fix them.

**Comment by ashleykleynhans** [comment]
NO region works fine, avoid FR region its broken.

**Comment by kozhemyak** [comment]
Yeah, I tried multiple regions: Canada, USA, and Netherlands. And they had the same issue. However, I read that it could be due to CPU configuration, I tried different Pod Sizes, and it worked.
Later that day, I tried to create 3090 in the USA again, and it started working as well.
I think there are some pods with CPU configuration that is not supported. It is random which pod you get.
Thank you. The question is closed.
